src/studio/src/designer/frontend/dashboard/features/createService/RepoTypeSelector.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+
+import AltinnInformationPaper from 'app-shared/components/AltinnInformationPaper';
+import { AltinnRadioGroup } from 'app-shared/components/AltinnRadioGroup';
+import { AltinnRadio } from 'app-shared/components/AltinnRadio';
+import { getLanguageFromKey } from 'app-shared/utils/language';
+
+import { useAppSelector } from 'common/hooks';
+import { DataModellingFormat } from 'services/repoApi';
+
+type RepoTypeSelectorProps = {
+  selectedFormat: DataModellingFormat;
+  onFormatChange: (newFormat: DataModellingFormat) => void;
+};
+
+const useStyles = makeStyles({
+  radioGroup: {
+    marginTop: '-2rem',
+  },
+});
+
+export const RepoTypeSelector = ({
+  selectedFormat,
+  onFormatChange,
+}: RepoTypeSelectorProps) => {
+  const classes = useStyles();
+  const language = useAppSelector((state) => state.language.language);
+
+  const handleChange = ({ target }: { target: HTMLInputElement }) => {
+    onFormatChange(target.value as DataModellingFormat);
+  };
+
+  return (
+    <div>
+      <Typography variant='h2'>
+        {getLanguageFromKey('dashboard.select_datamodelling_format', language)}
+      </Typography>
+      <AltinnRadioGroup
+        value={selectedFormat}
+        onChange={handleChange}
+        className={classes.radioGroup}
+      >
+        <AltinnRadio
+          label={getLanguageFromKey(
+            'dashboard.datamodelling_format_json',
+            language,
+          )}
+          value={DataModellingFormat.JSON}
+        />
+        <AltinnRadio
+          label={getLanguageFromKey(
+            'dashboard.datamodelling_format_xsd',
+            language,
+          )}
+          value={DataModellingFormat.XSD}
+        />
+      </AltinnRadioGroup>
+      <div style={{ margin: '12px 0 0 0' }}>
+        <AltinnInformationPaper>
+          <>
+            <p>
```

Comment:
Was wondering about the extra margin in the information papers here:
![image](https://user-images.githubusercontent.com/10416981/146344647-47da901e-563f-4ea3-8316-c8cc461e71fd.png)


I guess we have some global styles that add some margin to our paragraf which causes the difference in our inital margin in the paper. No big deal, but maybe get the spacing consistent? 🤔

Reply:
Oh, I didnt even notice at first. Ill fix!

- [ ] Fix incorrect spacing in infobox